csa/_elementary.py

```diff
@@ -125,8 +125,8 @@ def startIteration (self, state):
         obj.isPartitioned = False
         if 'partitions' in state:
             obj.isPartitioned = True
-            partitions = map (self.mask.intersection, state['partitions'])
-            sizes = map (len, partitions)
+            partitions = list(map (self.mask.intersection, state['partitions']))
```

Comment:
Please keep the original formatting style with space before argument list: list (...